src/core/discrete_space.jl

```diff
@@ -354,12 +354,21 @@ function get_node_contents(coords::Tuple, model::ABM{A,<:DiscreteSpace}) where {
 end
 
 """
-    get_node_agents(x, model)
+    get_node_agents(x, model) → agents
+
 Same as `get_node_contents(x, model)` but directly returns the list of agents
 instead of just the list of IDs.
+
+    get_node_agents(x::Array{Any, 1}, model) → agents
+
+Call `get_node_contents(x, model)` by broadcasting a list of nodes or agents.
+Useful to obtain a list of agents with their relative direction from starting 
+point `x`.
```

Comment:
Relative direction starting from point x is perhaps not clear enough. Are we going N, E, S, W for example?

Reply:
I think it's in line with the normal [von Neumann neighborhood](https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Von_Neumann_neighborhood.svg/1024px-Von_Neumann_neighborhood.svg.png) resulting in a `(N, W, E, S)` Tuple. I can add a quick note on that to the docstring.